src/NuGetPackageVerifier/Rules/AssemblyStrongNameRule.cs

```diff
@@ -33,7 +37,7 @@ public IEnumerable<PackageVerifierIssue> Validate(PackageAnalysisContext context
                     try
                     {
                         using (var packageFileStream = context.PackageReader.GetStream(currentFile))
-                        using (var fileStream = File.OpenWrite(assemblyPath))
+                        using (var fileStream = new FileStream(assemblyPath, FileMode.Create))
```

Comment:
To keep it consistent with [AssemblyAttributesDataHelper](https://github.com/aspnet/BuildTools/blob/2ffa222f550f01451bc3439b28ae0b01507a73a1/src/NuGetPackageVerifier/Rules/AssemblyAttributesDataHelper.cs#L36), but I can undo it.